sources/packages/web/src/views/student/StudentDashboard.vue

```diff
@@ -25,15 +45,43 @@
 import { computed } from "vue";
 import { useStore } from "vuex";
 import { StudentRoutesConst } from "../../constants/routes/RouteConstants";
-
+import { StudentService } from "@/services/StudentService";
+import { SetupContext, onMounted, ref } from "vue";
 export default {
-  setup() {
+  emits: ["update:formName", "change"],
+  props: {
+    formName: {
+      type: String,
+    },
+  },
+  setup(props: any, context: SetupContext) {
+    const programYearList = ref();
+    const onYearChange = (event: any) => {
+      context.emit("update:formName", event.value);
+      context.emit("change", event);
+    };
+    onMounted(async () => {
+      const programYears = await StudentService.shared.getProgramYears();
+      programYearList.value = programYears
+        ? programYears?.map((el: any) => ({
```

Comment:
Please consider changing this logic with the assumption that there will be aways a program year available. Without an active program year, the system is useless. It will simplify this logic a bit.

Reply:
I have seen many times that we are creating ".map(el" where "el" probably means "element".
I got a review about it mentioning that it would be better to have something specific instead of just "el".
Can we all follow that?
@ann-aot FYIA

Reply:
Please assign the correct type instead of "any" since it is available.

Reply:
done